.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,8 +1,8 @@
 name: ✨Feature Request
 description: Request a new feature or enhancement
-title: "[FEAT]: "
+title: ""
```

Comment:
o title n pode estar empty. TEns q remover mesmo.

Reply:
3792e0b should fix it